l10n_fr_fec_oca/wizard/account_fr_fec_oca.py

```diff
@@ -0,0 +1,579 @@
+# Copyright 2013-2020 Akretion France (http://www.akretion.com)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2016-2020 Odoo SA (https://www.odoo.com/fr_FR/)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+import base64
+import io
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from odoo.tools import float_is_zero
+import logging
+logger = logging.getLogger(__name__)
+
+try:
+    from unidecode import unidecode
+except ImportError:
+    logger.debug('Cannot import unidecode')
+try:
+    import unicodecsv
+except ImportError:
+    logger.debug('Cannot import unicodecsv')
+
+
+class AccountFrFecOca(models.TransientModel):
+    _name = 'account.fr.fec.oca'
+    _description = 'Ficher Echange Informatise'
+
+    date_range_id = fields.Many2one('date.range', string='Date Range')
+    date_from = fields.Date(string='Start Date', required=True)
+    date_to = fields.Date(string='End Date', required=True)
+    encoding = fields.Selection([
+        ('iso8859_15', 'ISO-8859-15'),
+        ('utf-8', 'UTF-8'),
+        # ('cp500', 'EBCDIC'),
+        ('ascii', 'ASCII'),
+        ], string='Encoding', default='iso8859_15', required=True)
+    delimiter = fields.Selection([
+        ('|', '|'),
+        ('tab', 'Tab'),
+        ], default='|', string='Field Delimiter', required=True)
+    partner_option = fields.Selection([
+        ('types', 'Selected Account Types'),
+        ('accounts', 'Selected Accounts'),
+        ('all', 'All'),
+        ], default='types', required=True, string='Partner Export Option')
+    partner_account_type_ids = fields.Many2many(
+        'account.account.type', string='Account Types',
+        default=lambda self: self._default_partner_account_type_ids())
+    partner_account_ids = fields.Many2many(
+        'account.account', string='Accounts',
+        default=lambda self: self._default_partner_account_ids())
+    fec_data = fields.Binary('FEC File', readonly=True, attachment=False)
```

Comment:
@alexis-via I wonder, why `attachment=False` ?
I believe it should be changed. Indeed, I had an issue in version 9 with this error : 
`File "/odoo/src/openerp/sql_db.py", line 233, in executeres = self._obj.execute(query, params)InternalError: invalid memory alloc request size 1073741824`
When Odoo try to write the binary into the database.
It seems the file was too big for postgresql... Storing in in the filestore resolve the issue. 

Of course, it only happen when we have a lot of data. 
Can we consider changing this ?

Reply:
I've put attachment=False because it is like that in the native module, cf https://github.com/odoo/odoo/blob/14.0/addons/l10n_fr_fec/wizard/account_fr_fec.py#L20
Feel free to change it if you think it is better.

Reply:
Done !